terraform/deployments/govuk-test/main.tf

```diff
@@ -65,4 +71,4 @@ module "govuk" {
   signon_desired_count              = var.signon_desired_count
   static_desired_count              = var.static_desired_count
   draft_static_desired_count        = var.draft_static_desired_count
-}
+}
```

Comment:
Somehow missing a line ending here apparently - might want to check your editor config.

Reply:
Wierd. Fixed.